sdk/appconfiguration/app-configuration/samples/getSettingOnlyIfChanged.ts

```diff
@@ -45,7 +45,7 @@ export async function run() {
 
 async function cleanupSampleValues(keys: string[], client: AppConfigurationClient) {
   const existingSettings = await client.listConfigurationSettings({
-    keys: keys
+    keyFilter: keys.join(',')
```

Comment:
Ah, thought this was part of the API but see it's part of a sample.  In which case I have no concerns.

Reply:
In fact it's what customers will likely end up doing :)